src/components/Inputs/checkbox/CheckBox.scss

```diff
@@ -0,0 +1,57 @@
+@import "src/theme/Theme.scss";
+
+.container {
+    display: block;
+    position: relative;
+    padding-left: 35px;
+    margin-bottom: 12px;
+    cursor: pointer;
+    font-size: 22px;
```

Comment:
22px? pisava mora bit ista kot body-m. Reši to tak da daš pravi classname checkboxu, ki že obstaja za pisavo.